### ISSUE TITLE
[Feature Request] Alternative mode for Active-Hint where active and inactive windows have different opacity

### ISSUE THREAD

**Issue by Konsonanz** [pop-os/shell#312]
Hello,
first off, enjoying your project quite a bit! I would like to state the following feature request:
- include the option to differentiate acitve and inactive windows by opacity

I am used to this behaviour from i3wm and would personally prefer it over the current option of indicating the active window with a colored border.

Thank you for your consideration!

**Comment by SoumyaRanjanNaik** [comment]
That would not be practical for me if the opacity of the window in which I have opened documentation changes as that will make it difficult to read and this would break the actual higher productivity of people if the opacity of an essential reference window changes.

**Comment by mmstick** [comment]
Of course, I don't see this being a default behavior, but it is something that we could add with minimal effort.

**Comment by SoumyaRanjanNaik** [comment]
If it is going to be an optional feature then it would actually be a good addition for people using it.
